app/assets/javascripts/analytics/custom-dimensions.js

```diff
@@ -46,6 +46,7 @@
       'political-status': {dimension: 7},
       'analytics:organisations': {dimension: 9},
       'analytics:world-locations': {dimension: 10},
+      'publication-status': {dimension: 12, defaultValue: 'other'},
```

Comment:
I believe publication-status is a separate thing to withdrawn.

Publication status would be:
* published
* force-published
* draft
* unpublished

Any of those could also be withdrawn/not-withdrawn.